docs/source/customizing.rst

```diff
@@ -1,6 +1,63 @@
 Customizing your BinderHub deployment
 =====================================
 
+Frontend
+--------
+
+Header and Footer customization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+BinderHub uses `Jinja <https://jinja.palletsprojects.com/en/stable/>`_ as template engine
+to process the page template `binderhub/templates/page.html <https://github.com/jupyterhub/binderhub/blob/main/binderhub/templates/page.html>`.
+To add a custom header and footer,
+
+1. copy ``binderhub/templates/page.html`` into ``files/custom-page.html``
+2. edit ``files/custom-page.html`` to include the desired header and footer.
+   For example::
+
+      <body>
+      <header>
+        My Own BinderHub
+      </header>
+      <div id="root"></div>
+      <footer>
+        Powered by BinderHub
+      </footer>
+      </body>
+3. add a `ConfigMap <https://kubernetes.io/docs/concepts/configuration/configmap/>`_ to your Helm configuration.
+   For example::
+
+    kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: binderhub-template-custom
+      labels:
+        app: binder
+        component: etc-binderhub
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+    data:
+      {{- (.Files.Glob "files/*").AsConfig | nindent 2 }}
+
```

Comment:
Is additional Helm configuration needed to mount the file in the ConfigMap into the BinderHub pod?